### DISCUSSION TITLE
Help with updating swizzin

### DISCUSSION THREAD

**Discussion by GitHub-BumBleBee** [swizzin/swizzin#1002] [Q&A + Help] [answered]
### What happened?

HELP

### Swizzin commit

HELP

### What OS are you using?

Ubuntu 22.04 (Jammy)

### What architecture is your OS?

amd64

### Relevant logs and output

```plaintext
I have 2 questions:
1. How do I see my swizzin version?


2. How do I upgrade from version 3.7.1 to version 3.8.0?
```

**Comment by liaralabs** [comment, marked as answer]
Issues are not for support purposes. They are to report bugs.

I can recommend several things:
1. [Read the docs](https://swizzin.ltd). This will probably answer the majority of any questions you may have.
2. Use the discord support to ask for help(after you've read the docs)
3. Use the discussions board here if you don't want to use discord.

**Comment by GitHub-BumBleBee** [comment]
I'm from Brazil and I can't find the information I need. Help me please!
